aztec/src/main/kotlin/org/wordpress/aztec/formatting/InlineFormatter.kt

```diff
@@ -265,7 +303,7 @@ class InlineFormatter(editor: AztecText, val codeStyle: CodeStyle, private val h
                 editableText.removeSpan(it)
             }
         }
-        // remove the CSS style span
```

Comment:
This appears to be another unwanted change that may have come about from merging from `trunk`. See also the deleted line at line 210 and the added line at 349 of this file.